src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    private String name;
```

Comment:
If you have `name` field in all Machines, maybe it's better to define it in parent Machine class?